src/50_Sample_AMPs/Live_Blog.html

```diff
@@ -0,0 +1,142 @@
+    <!-- #### Setup -->
+    <!--
+      All additionally used AMP components must be imported in the header.
+    -->
+    <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
+    <script async custom-element="amp-live-list" src="https://cdn.ampproject.org/v0/amp-live-list-0.1.js"></script>
+    </script>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <style amp-custom>
+  .carousel .slide > amp-img > img {
+  object-fit: contain;
+  }
+ .logo2 {
+   background-position: left 16px center;
+   background-repeat: no-repeat;
+   background-image: -webkit-image-set(
+     url(/img/ic_menu_white_1x_web_24dp.png) 1x,
+     url(/img/ic_menu_white_2x_web_24dp.png) 2x
+   );
+   background-color: #333;
+   text-align: left;
+   padding: 16px;
+   padding-left: 72px;
+   box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
+ }
+ .logo > a {
+   font-size: 16px;
+   font-weight: 500;
+   color: white;  
+   text-transform: uppercase;
+ }
+ .heading {
+   padding-bottom: 8px;
+ }
+ .heading > #summary {
+   font-weight: 500;
+ }
+ .heading > small {
+   color: #656565;
+ }
+
+ .heading .date {
+    color: #a8a3ae;
+    font-size: 12px;
+ }
+.blog {
+      padding: 0;
+      min-width: 70%;
+      background: #fff;
+}
+.blog p {
+      padding: 8px;
+      margin: 0;
+}
+
+.brand {
+  color: #6f757a;
+}
+.blog-item-info {
+  background: #f4f4f4;
+  border: solid #dcdcdc;
+  margin: 16px;
+}
+#live-list-update-button {
+  top: 60px;
+  position: fixed;
+  left: 50%;
+  transform: translateX(-50%);
+}
+  </style>
+
+  </head>
+  <body>
+
+    <!-- -->
+    <div class="logo">
+      <a href="/"> AMP by Example</a>
+    </div>
+
+    <div class="heading">
+      <h1>A sample Live Blog</h1>
+      <p><small class="date">18 July 2016</small></p>
+      <p id="summary">This is a sample article demonstrating how to write a live blog in AMP. It demonstrates the usage of AMP components which works well in news articles.</p>
```

Comment:
" It demonstrates the usage of AMP components which works well in news articles./"?